.test/config.yaml

```
@@ -1,6 +1,8 @@
 samples: samples.tsv
 units: units.tsv
 
+# trimming optional: yes or no
+trim: yes
```

I suggest:

```suggestion
adapter:
  trim: true
  sequence: ACGGATCGATCGATCGATCGAT
```

---

On the other hand, trimming might also involve non adapter sequences, right? E.g. quality trimming, depending on how people configure cutadapt. So maybe rather:

```suggestion
trimming:
  skip: false
  adapter: ACGGATCGATCGATCGATCGAT
```

---

ok new merge request now